products/rhcos4/profiles/stig-v1r1.profile

```diff
@@ -17,7 +17,7 @@ description: |-
     Red Hat Enterprise Linux CoreOS 4.
 
 selections:
-  - srg_ctr:all
+  - stig_ocp4:all
```

Comment:
In order to have a single control file for OCP4 STIG, the `rhcos4` profile references the `stig_ocp4` control file.
@rhmdnd @Vincent056 thoughts on this?

Reply:
That makes sense. Does this mean we can get rid of the separate `srg_gpos` and `srg_ctr` directories?

Reply:
Hmm, probably not.

As `srg_gpos` and `srg_ctr` are two different SRG publications they are better kept separate.

Before, where the `ocp4` and `rhcos4`profiles for selecting rules from `srg_ctr` control. Now they both source rules from `stig_ocp4` control.
The "innovation" here, and possible source of confusion, is that the control file is named `stig_ocp4`.